doc/api/http.md

```diff
@@ -729,8 +729,10 @@ This function is asynchronous. `callback` will be added as a listener for the
 
 Returns `server`.
 
-*Note*: The `server.listen()` method may be called multiple times. Each
-subsequent call will *re-open* the server using the provided options.
+*Note*:
+The `server.listen()` method can be called again iff there was an error
```

Comment:
I forgot what our position was on using 'iff' in the documentation: yay or nay?  `git grep` suggests nay.